ingredient/ingredient-databricks/src/arm.json

```diff
@@ -0,0 +1,104 @@
+        },
+        "workspaceName": {
+            "type": "string"
+        },
+        "tier": {
+            "defaultValue": "premium",
+            "type": "string"
+        },
+        "diagnosticsEnabled": {
+            "type": "string",
+            "defaultValue": "yes",
+            "metadata": {
+                "description": "Specifies whether to configure diagnostic settings to expose logs and metrics for consumption."
+            }
+        },
+        "diagnosticsSettingName": {
+            "type": "string",
+            "defaultValue": "diagToEHN",
+            "metadata": {
+                "description": "Diagnostic setting name.  Most by unique for the resource."
+            }
+        },
+        "diagnosticsEventHubResourceGroup": {
+            "type": "string",
+            "defaultValue": "",
+            "metadata": {
+                "description": "The resource group of the Event Hub Namespace where diagnostics are sent."
+            }
+        },
+        "diagnosticsEventHubNamespace": {
+            "type": "string",
+            "defaultValue": "",
+            "metadata": {
+                "description": "The Event Hub Namespace where diagnostics are sent."
+            }
+        },
+        "diagnosticsEventHubAuthorizationRule": {
+            "type": "string",
+            "defaultValue": "RootManageSharedAccessKey",
+            "metadata": {
+                "description": "The authorization rule to use for transmitting diagnostics."
+            }
+        }
+    },
+    "resources": [
+        {
+            "type":  "Microsoft.Databricks/workspaces",
+            "apiVersion": "2018-04-01",
+            "location": "[parameters('location')]",
+            "name": "[parameters('workspaceName')]",
+            "sku": {
+                "name": "[parameters('tier')]"
+            },
+            "comments": "Please do not use an existing resource group for ManagedResourceGroupId.",
+            "properties": {
+                "ManagedResourceGroupId": "[variables('managedResourceGroupId')]"
+            },            
+            "resources": [
+                {
+                    "condition": "[equals(parameters('diagnosticsEnabled'), 'yes')]",
+                    "type": "providers/diagnosticSettings",
+                    "name": "[concat('Microsoft.Insights/', parameters('diagnosticsSettingName'))]",
+                    "dependsOn": [                    
+                    "[parameters('workspaceName')]"
+                    ],
+                    "apiVersion": "2017-05-01-preview",
+                    "properties": {
+                    "name": "[parameters('diagnosticsSettingName')]",
+                    "eventHubAuthorizationRuleId": "[resourceId(parameters('diagnosticsEventHubResourceGroup'),'Microsoft.EventHub/namespaces/authorizationRules', parameters('diagnosticsEventHubNamespace'), parameters('diagnosticsEventHubAuthorizationRule'))]",
+                    "logs": [
```

Comment:
Can you confirm that you only want workspace and jobs logs?  I see that there are others available when looking at the list in Azure for Databricks diagnostic settings but am not certain if they would be valuable in troubleshooting scenarios.